src/epub/templates/song.css

```diff
@@ -0,0 +1,127 @@
+h1 {
+  text-align: left
+  }
+
+span.lyric span.ch {
+  display: inline-block;
+  position: relative;
+  left: 0px;
+  top: -1.4em;
+  font-size: xx-small;
+  width:0px;
+  padding-top:0.9em;
+  font-weight: bold;
+  overflow-wrap: normal;
+  hyphens: none;
+}
+
+span.lyric span.akord {
```

Comment:
akord ->  let's be consistent and use english here.

Reply:
done